setup.py

```diff
@@ -282,6 +292,30 @@ def report(*args):
 
 cmake = CMake()
 
+# setup environment variables for emulation build
+def setup_emul():
+    # here we can't just set DEBUG=1, because build_type is imported
+    # at the beginning of this file ... at that moment DEBUG is unset
+    # and build_type will be release
+    os.environ['CMAKE_BUILD_TYPE'] = 'Debug'
+    os.environ["BUILD_TEST"] = "0"
+    os.environ["USE_MKL"] = "0"
+    os.environ["USE_MKLDNN"] = "0"
+    os.environ["USE_CUDA"] = "0"
+    os.environ["USE_CUDNN"] = "0"
+    os.environ["USE_FBGEMM"] = "0"
+    os.environ["USE_NNPACK"] = "0"
+    os.environ["USE_QNNPACK"] = "0"
+    os.environ["USE_DISTRIBUTED"] = "0"
+    os.environ["USE_OPENMP"] = "0"
+    os.environ["ATEN_THREADING"] = "NATIVE"
+    os.environ["OMP_NUM_THREADS"] = "1"
+    os.environ["USE_HB_EMUL"] = "1"
+    if os.getenv("BSG_MANYCORE_DIR", "") != "":
+        del os.environ["BSG_MANYCORE_DIR"]
+    if os.getenv("HB_KERNEL_DIR", "") != "":
+        del os.environ["HB_KERNEL_DIR"]
```

Comment:
Couldn't understand this part... where do we set these variables?

Reply:
Precautions. Same as
https://github.com/cornell-brg/hb-pytorch/blob/413850923789b97b317edf43585d62e6a50d119e/setup_emul_build_env.sh#L29-L31

Reply:
Oh, sorry, this is emulation, we don't need them... my bad!